packages/core/metrics/core-metrics-server/src/metrics.ts

```diff
@@ -40,6 +40,46 @@ export interface IntervalHistogram {
   };
 }
 
+/**
+ * Metrics related to the elasticsearch clients
+ * @public
+ */
+export interface ElasticsearchClientsMetrics {
+  /** Number of HTTP Agents that have are currently being used by the ES-js client */
+  agents: number;
+  /** Number of ES instances (or proxies) that ES-js client is connecting to */
```

Comment:
NIT:
```suggestion
  /** Number of ES nodes that ES-js client is connecting to */
```
Kibana doesn't support connecting  through a proxy.